lib/bfd.h

```diff
@@ -51,9 +51,10 @@ struct bfd_gbl {
 #define BFD_FLAG_BFD_CBIT_ON (1 << 3) /* Peer registered with CBIT set to on */
 #define BFD_FLAG_BFD_CHECK_CONTROLPLANE (1 << 4) /* BFD and controlplane daemon are linked */
 
-#define BFD_STATUS_UNKNOWN (1 << 0) /* BFD session status never received */
-#define BFD_STATUS_DOWN    (1 << 1) /* BFD session status is down */
-#define BFD_STATUS_UP      (1 << 2) /* BFD session status is up */
+#define BFD_STATUS_UNKNOWN    (1 << 0) /* BFD session status never received */
+#define BFD_STATUS_DOWN       (1 << 1) /* BFD session status is down */
+#define BFD_STATUS_UP         (1 << 2) /* BFD session status is up */
+#define BFD_STATUS_ADMIN_DOWN (1 << 3) /* BFD session is administatrively down */
```

Comment:
We'll need an update to `lib/bfd.c:bfd_get_status_str()` also

Reply:
@mjstapp 
Current protocols store only UP/DOWN state so this API in current state is sufficient.
Anyways i will update the code, it is good to have all the states updated.

Reply:
so ... at least ospf, I think, calls the `str` api with the raw value - just saying.

Reply:
Updated